uw-frame-components/portal/notifications/directives.js

```diff
@@ -11,16 +11,17 @@ define(['angular', 'require'], function(angular, require){
         }
     });
 
-    app.directive('notificationBell', function(){
+    app.directive('notificationPartials', function(){
```

Comment:
That seems like a good idea, although I think 'ap' might be confusing - looks too much like the word 'app'. But some kind of semantic namespacing would be good. 

Yea, I suppose changing that directive name is a breaking change - good catch. Should we stick with what it was before until 3.0.0? And at that point we can talk about namespacing directives?